src/canvas_renderer.js

```diff
@@ -199,7 +199,7 @@ app.CanvasRenderer = class {
             ctx.save();
             ctx.beginPath();
```

Comment:
we don't need the whole thing - starting from ctx.save() and ending with ctx.restore() (the drawImage should stay)